Index: devel/rcs/files/patch-src__b-complain.h
===================================================================
--- devel/rcs/files/patch-src__b-complain.h	(revision 0)
+++ devel/rcs/files/patch-src__b-complain.h	(working copy)
@@ -0,0 +1,20 @@
+--- src/b-complain.h.orig	2013-04-20 20:08:38.000000000 +0200
++++ src/b-complain.h	2015-02-24 22:01:39.465329000 +0100
+@@ -29,12 +29,11 @@
+   printf_string (2, 3);
+ extern void generic_error (char const *who, char const *fmt, ...)
+   printf_string (2, 3);
+-extern void generic_fatal (char const *who, char const *fmt, ...)
+-  printf_string (2, 3) exiting;
+-extern void fatal_syntax (size_t lno, char const *fmt, ...)
+-  printf_string (2, 3) exiting;
+-extern void fatal_sys (char const *who)
+-  exiting;
++extern exiting void generic_fatal (char const *who, char const *fmt, ...)
++  printf_string (2, 3);
++extern exiting void fatal_syntax (size_t lno, char const *fmt, ...)
++  printf_string (2, 3);
++extern exiting void fatal_sys (char const *who);
+ 
+ /* Idioms.  Here, prefix P stands for "program" (general operation);
+    M for "manifestation"; R for "repository".  */

Property changes on: devel/rcs/files/patch-src__b-complain.h
___________________________________________________________________
Added: fbsd:nokeywords
## -0,0 +1 ##
+yes
\ No newline at end of property
Added: svn:eol-style
## -0,0 +1 ##
+native
\ No newline at end of property
Added: svn:mime-type
## -0,0 +1 ##
+text/plain
\ No newline at end of property
Index: devel/rcs/files/patch-src__b-fb.h
===================================================================
--- devel/rcs/files/patch-src__b-fb.h	(revision 0)
+++ devel/rcs/files/patch-src__b-fb.h	(working copy)
@@ -0,0 +1,14 @@
+--- src/b-fb.h.orig	2013-04-20 20:08:38.000000000 +0200
++++ src/b-fb.h	2015-02-24 21:58:19.888149000 +0100
+@@ -21,9 +21,9 @@
+ */
+ 
+ extern int change_mode (int fd, mode_t mode);
+-extern void Ierror (void) exiting;
++exiting extern void Ierror (void);
+ extern void testIerror (FILE *f);
+-extern void Oerror (void) exiting;
++exiting extern void Oerror (void);
+ extern void testOerror (FILE *o);
+ extern FILE *fopen_safer (char const *filename, char const *type);
+ extern void Ozclose (FILE **p);

Property changes on: devel/rcs/files/patch-src__b-fb.h
___________________________________________________________________
Added: fbsd:nokeywords
## -0,0 +1 ##
+yes
\ No newline at end of property
Added: svn:eol-style
## -0,0 +1 ##
+native
\ No newline at end of property
Added: svn:mime-type
## -0,0 +1 ##
+text/plain
\ No newline at end of property
Index: devel/rcs/files/patch-src__base.h
===================================================================
--- devel/rcs/files/patch-src__base.h	(revision 0)
+++ devel/rcs/files/patch-src__base.h	(working copy)
@@ -0,0 +1,22 @@
+--- src/base.h.orig	2013-10-20 10:48:18.000000000 +0200
++++ src/base.h	2015-02-24 21:57:04.059371000 +0100
+@@ -753,8 +753,7 @@
+ int donerewrite (int changed, time_t newRCStime);
+ void ORCSclose (void);
+ void ORCSerror (void);
+-void unexpected_EOF (void)
+-  exiting;
++exiting void unexpected_EOF (void);
+ void initdiffcmd (struct diffcmd *dc);
+ int getdiffcmd (struct fro *finfile, bool delimiter,
+                 FILE *foutfile, struct diffcmd *dc);
+@@ -829,8 +828,7 @@
+                       char datebuf[datesize + zonelenmax]);
+ 
+ /* rcsutil */
+-void thank_you_and_goodnight (int const how)
+-  exiting;
++exiting void thank_you_and_goodnight (int const how);
+ /* These are for ‘thank_you_and_goodnight’.  */
+ #define TYAG_ORCSERROR     (1 << 3)
+ #define TYAG_DIRTMPUNLINK  (1 << 2)

Property changes on: devel/rcs/files/patch-src__base.h
___________________________________________________________________
Added: fbsd:nokeywords
## -0,0 +1 ##
+yes
\ No newline at end of property
Added: svn:eol-style
## -0,0 +1 ##
+native
\ No newline at end of property
Added: svn:mime-type
## -0,0 +1 ##
+text/plain
\ No newline at end of property
--- rcs-5.9.4/src/base.h.omv~	2015-07-20 12:55:06.713084180 +0200
+++ rcs-5.9.4/src/base.h	2015-07-20 12:55:34.995166957 +0200
@@ -754,8 +754,7 @@ int dorewrite (bool lockflag, int change
 int donerewrite (int changed, time_t newRCStime);
 void ORCSclose (void);
 void ORCSerror (void);
-void unexpected_EOF (void)
-  exiting;
+void exiting unexpected_EOF (void);
 void initdiffcmd (struct diffcmd *dc);
 int getdiffcmd (struct fro *finfile, bool delimiter,
                 FILE *foutfile, struct diffcmd *dc);
@@ -830,8 +829,7 @@ char const *date2str (char const date[da
                       char datebuf[datesize + zonelenmax]);
 
 /* rcsutil */
-void thank_you_and_goodnight (int const how)
-  exiting;
+void exiting thank_you_and_goodnight (int const how);
 /* These are for ‘thank_you_and_goodnight’.  */
 #define TYAG_ORCSERROR     (1 << 3)
 #define TYAG_DIRTMPUNLINK  (1 << 2)
--- rcs-5.9.4/src/b-complain.h.omv~	2015-07-20 12:55:53.278219219 +0200
+++ rcs-5.9.4/src/b-complain.h	2015-07-20 12:56:33.701331480 +0200
@@ -32,12 +32,11 @@ extern void generic_warn (char const *wh
   printf_string (2, 3);
 extern void generic_error (char const *who, char const *fmt, ...)
   printf_string (2, 3);
-extern void generic_fatal (char const *who, char const *fmt, ...)
-  printf_string (2, 3) exiting;
-extern void fatal_syntax (size_t lno, char const *fmt, ...)
-  printf_string (2, 3) exiting;
-extern void fatal_sys (char const *who)
-  exiting;
+extern void exiting generic_fatal (char const *who, char const *fmt, ...)
+  printf_string (2, 3);
+extern void exiting fatal_syntax (size_t lno, char const *fmt, ...)
+  printf_string (2, 3);
+extern void exiting fatal_sys (char const *who);
 
 /* Idioms.  Here, prefix P stands for "program" (general operation);
    M for "manifestation"; R for "repository".  */
--- rcs-5.9.4/src/b-excwho.c.omv~	2015-07-20 12:57:15.546443221 +0200
+++ rcs-5.9.4/src/b-excwho.c	2015-07-20 12:57:55.499545990 +0200
@@ -123,7 +123,7 @@ set_uid_to (uid_t u)
       if (BE (already_setuid))
         return;
       BE (already_setuid) = true;
-      PFATAL ("root setuid not supported" + (u ? 5 : 0));
+      PFATAL ("%s", "root setuid not supported" + (u ? 5 : 0));
     }
 }
 #endif  /* defined HAVE_SETUID */
--- rcs-5.9.4/src/b-fb.h.omv~	2015-07-20 12:58:07.767576824 +0200
+++ rcs-5.9.4/src/b-fb.h	2015-07-20 12:58:29.614630934 +0200
@@ -21,9 +21,9 @@
 */
 
 extern int change_mode (int fd, mode_t mode);
-extern void Ierror (void) exiting;
+extern void exiting Ierror (void);
 extern void testIerror (FILE *f);
-extern void Oerror (void) exiting;
+extern void exiting Oerror (void);
 extern void testOerror (FILE *o);
 extern FILE *fopen_safer (char const *filename, char const *type);
 extern void Ozclose (FILE **p);
